content/docs/csidriver/installation/operator/_index.md

```diff
@@ -405,7 +405,7 @@ spec:
 You can set the field ***replicas*** to a higher number than `1` for the latest driver versions.
 
 Note - The `image` field should point to the correct image tag for version of the driver you are installing.  
-For e.g. - If you wish to install v2.5.0 of the CSI PowerMax driver, use the image tag `dellemc/csi-powermax:v2.5.0`
+For e.g. - If you wish to install v2.6.0 of the CSI PowerMax driver, use the image tag `dellemc/csi-powermax:v2.6.0`
```

Comment:
why there Powrmax driver mention here ? any reason ?

Reply:
@prablr79 - This content is already present in the documentation and it is for an example. Please Refer. https://dell.github.io/csm-docs/docs/csidriver/installation/operator/
![image](https://user-images.githubusercontent.com/43825650/220534676-ecc65e7c-e5ad-44dd-a719-8d4d26f3d320.png)

Reply:
let us change Eg to Pscale. Thanks!

Reply:
@prablr79 : It is operator generic page, not Isilon page